guess-the-country/country-frontend/README.md

```diff
@@ -24,4 +23,16 @@ npm run unit
 npm test
 ```
 
+## Environment Variables
+VUE_APP_BACKEND_URL=""
+VUE_APP_PUBLIC_PATH="/"
+VUE_APP_IMAGE_SEQUENCE_MODE='classic'
+
+| Variable    | Description                                               | Default Value |
+|-------------|-----------------------------------------------------------|---------------|
+| `VUE_APP_BACKEND_URL` | Backend url                                               | `""`          |
+| `VUE_APP_PUBLIC_PATH`          | Public path                                               | `"/"`         |
```

Comment:
@ionicsolutions Fällt dir eine bessere Beschreibung ein?

Reply:
Das ist nicht so toll beschrieben, da hast du recht. aber das gilt für alle Use Cases und muss nicht in diesem PR "gelöst" werden.